src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -472,7 +484,10 @@ public static void WebSocket_Http_Duplex_TextStreamed()
 
     [Fact]
     [OuterLoop]
+#if FULLXUNIT_NOTSUPPORTED
+#else
     [ActiveIssue(420, PlatformID.AnyUnix)]
+#endif
```

Comment:
ToF does not understand conditional ActiveIssues so it just treats this as an unconditional ActiveIssue and skips it.